src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -199,7 +199,8 @@ def make_parameter_file(
         azimuth_looks: int,
         range_looks: int,
         dem_name: str = 'GLO_30',
-        dem_resolution: int = 30) -> None:
+        dem_resolution: int = 3,
```

Comment:
Looks like the `0` got dropped.
```suggestion
        dem_resolution: int = 30,
```